docs/configuration/extract-codacy-logs.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+
+#### Helper functions ####
+
+usage()
+{
+  echo "Usage: $0 -n NAMESPACE "
+  exit 2
+}
+
+set_variable()
+{
+  local varname=$1
+  shift
+  if [ -z "${!varname}" ]; then
+    eval "$varname=\"$@\""
+  else
+    echo "Error: $varname already set"
+    usage
+  fi
+}
+
+cleanup()
+{
+	if [ -z $LOGS_DIR ]; then
```

Comment:
Why is the indentation here different? 
Can you add an `.editorconfig` yo the root of this project?